### DISCUSSION TITLE
FPS drops a lot (to 1 FPS) when I create a lobby

### DISCUSSION THREAD

**Discussion by Nekrapiwa** [FakeFishGames/Barotrauma#13685] [Bug reports]
Hello!

When creating the server, the first 30 seconds are fine. After that, it starts to drop sharply down to 1 fps.

I've been playing this game for over 80 hours now. I encountered this problem for the first time yesterday. Before this situation everything was fine.

The problem began with the fact that on July 15, 2023 I played with friends for several hours. All of a sudden my FPS dropped to 1. The rest of the players had the same. It's just some kind of slideshow. I can not do anything.

I tried to play many other games (including online games). Everything was good as always. The computer works well and stably, the Internet too. This problem is only with Barotrauma.

### WHAT DOES IT LOOK LIKE:
Pay attention to the graph called "Update"

1. I am creating a server lobby. All is well for the first ~20-30 seconds

![1](https://github.com/Regalis11/Barotrauma/assets/139697180/7e531003-8dee-48b4-a869-1f0f53300f2d)

2. Something starts to happen

![2](https://github.com/Regalis11/Barotrauma/assets/139697180/833bd733-63dd-43b6-a50f-f55a605ed227)

3. This is starting to intensify. FPS sharply strive to 1

![3](https://github.com/Regalis11/Barotrauma/assets/139697180/9ba0e9d7-2340-4cbf-b0bf-10978198f26a)

4. The situation worsens every second to infinity. This will only stop when I close the game and restart when I open the server. 
FPS ranges from 1 to 3-4

![4](https://github.com/Regalis11/Barotrauma/assets/139697180/69fff33c-b5ca-4cee-a0a3-d75be2defb63)

**What I have tried:**
1. I have updated all drivers and updates on my PC.
2. Reinstalled the game several times
3. Used various settings inside the game
4. Disabled steam overlay and cloud saves
5. Turned off antivirus and firewall.
6. Restart PC and internet router

The problem still remains. Help me please.
I like this game and I want to play it.

My pc:
2060 12gb
amd ryzen 5600x
16 gb ddr4

**Comment by NilanthAnimosus** [comment]
Thanks for the report!

I've tested this against 1.0.21.0 and Lua for barotrauma in case on the same version and for both I could not get these results (or any increase at all) while letting it run for over 5 minutes in each try.

You did not use the issue template, so I don't know the operating system, if mods are used, what game version exactly (Unstable?), Or any other potential hints to go off in order to try to help figure out this issue.

The hardware looks to be fine, and I see no issues there.

Do you still get this issue on the current unstable as well as the current version of the game? Do you perhaps have any other ideas as to what could be causing it?
